vendor/github.com/bitrise-io/go-utils/command/git/commands.go

```diff
@@ -162,3 +162,10 @@ func (g *Git) SparseCheckoutSet(opts ...string) *command.Model {
 	args = append(args, opts...)
 	return g.command(args...)
 }
+
```

Comment:
How did this update if there is no go.sum change? Is the vendor folder consistent?

Reply:
Updated with correct vendor updates.